ironfish/src/workerPool/tasks/verifyTransactions.test.ts

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This was copied from `verifyTransaction.test.ts` and slightly refactored for verifyTransactions.